src/librustc/hir/lowering.rs

```diff
@@ -5360,7 +5268,7 @@ impl<'a> LoweringContext<'a> {
             Some(id) => (id, "`'_` cannot be used here", "`'_` is a reserved lifetime name"),
 
             None => (
-                self.next_id().node_id,
+                self.sess.next_node_id(),
```

Comment:
This no longer lowers the node id.

Reply:
`next_id` calls `lower_node_id` with `sess.next_node_id()`; then, `lower_node_id` just passes that one on to `lower_node_id_generic` that returns that same initial `node_id`, so this results in the same outcome as before, doesn't it?

Or do you mean that `self.node_id_to_hir_id.resize(min_size, hir::DUMMY_HIR_ID);` from `lower_node_id_generic` is no longer called?

Reply:
I see the node id gets lowered in `new_named_lifetime` below, so this doesn't matter.